css/_videolayout_default.scss

```diff
@@ -44,13 +44,20 @@
 .videocontainer {
     position: relative;
     text-align: center;
+
+    &__wrapper {
```

Comment:
If I'm understanding correctly this is only for the black background, right? Maybe we should name the class `__background`. WDYT?